delivery_price_rule_volumetric_weight/README.rst

```diff
@@ -39,7 +39,7 @@ To configure this module, you need to:
 
 #. Go to *Sale -> Configuration -> Shipping Methods*.
 #. Create a new Shipping method or edit some record and set Based on rule providers.
-#. Add new rule and set Volumetric weight in condition and in variable factor.
+#. Add new rule and set Volumetric weight or Ponderate weight in condition and in variable factor.
```

Comment:
I'm not sure that **ponder** is the proper english term for this (https://www.merriam-webster.com/dictionary/pondered). It would be **pondered** in any case, as **ponderate** means **deliberate** (https://www.merriam-webster.com/dictionary/ponderate#dictionary-entry-2)

Reply:
ping @LudLaf

Reply:
"Ponderate" has a meaning that doesn't fit in this description. An option could be "weighted average". Only thing is, it'll seem to be a redundancy ("weighted average weight"), but probably "weighted average load" instead of "Ponderate weight" will help.

ping @victoralmau @chienandalu

Reply:
What we want to express is the maximum between the volumetric weight and the normal weight, but in the most condensed way. What a challenge... Any suggestion?

Reply:
What about modulated_weight?

Reply:
What about "higher weight"? Quite simple, and suggests the biggest option between volumetric weight and physical weight.

Reply:
> What about "higher weight"?

Wouldn't it be highest weight?

Reply:
Sure, highest weight.